src/tests/Common/CLRTest.Execute.Bash.targets

```diff
@@ -407,12 +407,7 @@ else
 	HARNESS_RUNNER="xharness"
 fi
 
-xcode_path="%24(dirname "%24(dirname "%24(xcode-select -p)")")"
-simulator_app="$xcode_path/Contents/Developer/Applications/Simulator.app"
-open -a "$simulator_app"
-helix_runner_uid="%24(id -u)"
-
-sudo launchctl asuser "$helix_runner_uid" $__Command $HARNESS_RUNNER apple just-run %5c
+$__Command $HARNESS_RUNNER apple just-run %5c
```

Comment:
We need to make the test target below dynamic:
```
  --targets ios-simulator-64 %5c
```

Reply:
Will be resolved via a future PR. Created an issue to track this https://github.com/dotnet/runtime/issues/59185